utils/crates-io/src/manifest.rs

```diff
@@ -0,0 +1,47 @@
+//! Manifest utils for crates-io-manager
+
+use anyhow::Result;
+use cargo_toml::{Manifest, Value};
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+
+/// Cargo manifest with path
+pub struct ManifestWithPath {
+    /// Crate name
+    pub name: String,
+    /// Cargo manifest
+    pub manifest: Manifest,
+    /// Path of the manifest
+    pub path: PathBuf,
+}
+
+impl ManifestWithPath {
+    /// Get the worksapce manifest
+    pub fn workspace() -> Result<Self> {
+        let path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
```

Comment:
maybe use `std::env` instead of `env!`?

Reply:
`std::env` doesn't work here bcz it is for execution, instead `env!` for compilation which is totally safe here https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates